sechub-analyzer-cli/src/main/java/com/daimler/analyzer/model/AnalyzerResult.java

```diff
@@ -0,0 +1,47 @@
+package com.daimler.analyzer.model;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * A container class for the analysis result.
+ */
+public class AnalyzerResult {
+
+    private Map<String, List<MarkerPair>> findings;
+
+    public AnalyzerResult(Map<String, List<MarkerPair>> findings) {
+        this.findings = findings;
+    }
+    
+    public Map<String, List<MarkerPair>> getFindings() {
```

Comment:
Please rename to `getNoSecHubMarkers()`

Reply:
As long as we do not provide this as an API it's okay to provide this map directly over the method (I have done this as well at some sechub locations). But we must be aware when this is used at different locations this can cause trouble - reason: The map is not immutable. but directly returned, so callers can change the content!